File: Ajay_thakur/Assignments/18_01_2025/06.java
/*
 * WAP to check if a given string contain special character or not
 */

import java.util.*;

class Main{

    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter your girlfriend name");

        String s = sc.nextLine();

        int flag = 0;

        for(int i=0;i<s.length();i++){

            if(s.charAt(i)=='$'||s.charAt(i)=='@'||s.charAt(i)=='#'){

                flag = 1;
                break;

            }
        }

        if(flag==1){

            System.out.println("given String contain Special character");
        } else{

            System.out.println("not present");
        }

        
    }
}

File: Javascript/Assignment/06_05_2025/06.js
let str = "i love$() coding@";

function special(s){

    let count = 0;
    for(i=0;i<s.length;i++){

        let ch = s.charAt(i);

        if(ch>='A' && ch<='Z' || ch>='a' && ch<='z' || ch>='0' && ch<='9' || ch==' '){

            console.log(ch);
            continue;

        }else{

            console.log(ch);
            count ++;
        }
    }

    if(count>0){

        return `special character present count is ${count}`;
    } else{

        return "no special character present";
    }

    
}

console.log(special(str)) ;